widgets/opal/commons/org.eclipse.nebula.widgets.opal.commons/src/org/eclipse/nebula/widgets/opal/commons/ReflectionUtils.java

```diff
@@ -0,0 +1,42 @@
+package org.eclipse.nebula.widgets.opal.commons;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class ReflectionUtils {

```

Comment:
Missing javadoc for API

Reply:
Ok. Should I fix it before the XMas release or could I work on this next week ?